components/components/view-modes/highlighted-small/_node-highlighted-small.css

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file
+ * This file is used to style the 'highlighted-small' view mode.
+ */
+
+/*
+  Highlighted Small
+
+  .node--view-mode-highlighted-small - This presents the 'Highlighted Small' view mode.
```

Comment:
This kind of comment (class or pseudo class name with comment) called to "modifier" in KSS and this is what duplicates the markup in the generated styleguide. It is not needed here. (Should be used eg. for different button states.)

Reply:
I'm not sure I follow you Thamas. I have changed it to just "This presents the 'Highlighted Small' view mode."

is that ok?